### PR TITLE
Add module files to compile on AWS

### PR DESCRIPTION
**Pull request by weihuang-jedi**
Add module files to compile on AWS

Solve issue: #1081 

https://github.com/NOAA-EMC/GDASApp/issues/1081

- Tested on AWS
- Tested on Hera

### CODE REVIEW

File: ush/module-setup.sh [hunk 1]
```diff
@@ -105,6 +105,10 @@ elif [[ $MACHINE_ID = discover* ]]; then
     export PATH=$PATH:$SPACK_ROOT/bin
     . $SPACK_ROOT/share/spack/setup-env.sh
 
+elif [[ $MACHINE_ID = noaacloud* ]]; then
+    # We are on NCCS discover
```
Comment (aerorahul) [inline ush/module-setup.sh]:
```suggestion
    # We are on NOAA Cloud
```

**Review by aerorahul — CHANGES_REQUESTED** [review]
Just a minor comment.

File: ush/module-setup.sh (hunk 1 above)
Comment (weihuang-jedi) [reply ush/module-setup.sh]: Changed.

**Review by aerorahul — APPROVED** [review]
looks good

**Comment by weihuang-jedi** [comment]
Thank you @Rahul Mahajan - NOAA Federal ***@***.***> .

**Review by CoryMartin-NOAA — APPROVED** [review]

**Comment by weihuang-jedi** [comment]
Thank you, Cory!

**Pull request merged**